src/test/ui/associated-types/bound-region-variant-resolution.rs

```diff
@@ -0,0 +1,15 @@
+// This is a bug!
```

Comment:
can you open an issue for that? You can then add `// known-bug: #<issuenum>` to this file